examples/typescript/aws-s3-static-website/main.ts

```diff
@@ -0,0 +1,98 @@
+import { Construct } from "constructs";
+import { App, TerraformStack, TerraformOutput, TerraformVariable } from "cdktf";
+import { AwsProvider, s3 } from '@cdktf/provider-aws';
```

Comment:
We run examples in CI to ensure they still work with new versions of the CDKTF that we might release, which includes changes to provider bindings for some releases. This means, we cannot use pre-built providers in examples, as they are only available after a CDKTF version has been released. So we'd need to switch this example to use locally built providers (also requires adding the AWS Provider to the `cdktf.json` config). Refer to [this example](https://github.com/hashicorp/terraform-cdk/tree/main/examples/typescript/aws-cloudfront-proxy) for import paths, etc.

Reply:
Done!